tests/unit/prop-types/null-test.js

```diff
@@ -64,6 +173,60 @@ describe('Unit / validator / PropTypes.null', function () {
       })
 
       itValidatesTheProperty(ctx, 'Missing required property bar')
+
+      describe('when updated with array value', function () {
```

Comment:
Is it me, or are these really repetative? Could we make this DRYer by having something like an object of values to set it to and expected error messages?